packages/canon/scripts/build-css.mjs

```diff
@@ -58,23 +58,53 @@ if (!fs.existsSync(distDir)) {
   fs.mkdirSync(distDir, { recursive: true });
 }
 
-// Bundle and transform files
-cssFiles.forEach(file => {
-  let { code: bundleCode } = bundle({
-    filename: file.path,
-  });
+// Add watch mode support
+const args = process.argv.slice(2);
+const watchMode = args.includes('--watch');
+
+async function buildCSS(logs = true) {
+  // Bundle and transform files
+  cssFiles.forEach(file => {
+    let { code: bundleCode } = bundle({
+      filename: file.path,
+    });
+
+    let { code, map } = transform({
```

Comment:
We could do this kinda thing to prettify the code after transform: https://github.com/backstage/backstage/blob/547e41da5ac497e5a606c08c8fb57b429687d5f7/packages/repo-tools/src/commands/api-reports/common/tryRunPrettier.ts#L25-L31

That was we can keep the CSS pretty and avoid the prettier ignore

Reply:
The idea is to keep it minified in production. The CSS is quite extensive so this will help loading time. I'm not sure we will ever have to actually read the file directly. The minified process doesn't change any names so if anyone want to read it they can prettify it pretty easily.

Reply:
it will be minified in production later on when the app is built anyway, we don't ship minimized JS code either

Reply:
Ok. I removed the minify step